server/sonar-server/src/main/java/org/sonar/server/branch/pr/ws/ListAction.java

```diff
@@ -112,13 +113,12 @@ private static void addPullRequest(ProjectPullRequests.ListWsResponse.Builder re
     Optional<BranchDto> mergeBranch = Optional.ofNullable(mergeBranchesByUuid.get(branch.getMergeBranchUuid()));
     ProjectPullRequests.PullRequest.Builder builder = ProjectPullRequests.PullRequest.newBuilder();
     String branchKey = branch.getKey();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "getPullRequestData()" can return null. ([squid:S2259](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2259))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjanos%2FSONAR-10365%2Fuse-pr-id-as-key-for-pull-requests&issues=AWG35kc9JosG6VNHvO3E&open=AWG35kc9JosG6VNHvO3E)

[ISSUE_KEY]: AWG35kc9JosG6VNHvO3E